apps/src/p5lab/gamelab/GameLab.js

```diff
@@ -1,9 +1,23 @@
 import P5Lab from '../P5Lab';
+import project from '@cdo/apps/code-studio/initApp/project';
+import {showLevelBuilderSaveButton} from '../../code-studio/header';
 
 var GameLab = function() {
   P5Lab.call(this);
 };
 
 GameLab.prototype = Object.create(P5Lab.prototype);
 
+GameLab.prototype.init = function(config) {
+  if (config.level.editBlocks) {
```

Comment:
TODO: find a better method for editing starting libraries and blocks in gamelab.
https://codedotorg.atlassian.net/browse/STAR-804